src/Entity/Board.php

```diff
@@ -451,4 +451,14 @@ public function setHearingPostResponseTemplate(?MailTemplate $hearingPostRespons
 
         return $this;
     }
+
+    public function getFinalStatus(): string
+    {
+        $rawPlaces = explode(
+            PHP_EOL,
+            trim($this->getStatuses())
+        );
```

Comment:
Use 

```php
        $rawPlaces = array_filter(
            array_map(
                'trim',
                explode(
                    PHP_EOL,
                    $this->getStatuses()
                )
            )
        );
```

to make this work as expected, i.e. to trim all lines after exploding and retain only non-empty lines.

Reply:
👍